### PR TITLE
Edit analytics data from object to string, add discussion log events

### PR DESCRIPTION
**Pull request by mcbouslog**
Staging branch URL: https://add-easy-sw-analytics.pfe-preview.zooniverse.org/

Towards requested Snapshot Wisconsin analytics requests.

- add log for viewing Discussion
- add log for new Discussion title
- add log boolean in TalkQuickSubjectCommentForm's CommentBox component, to enable logging 
- edit SW requested log events data from objects to strings

# Required Manual Testing

- [x] Does the non-logged in home page render correctly?
- [x] Does the logged in home page render correctly?
- [x] Does the projects page render correctly?
- [x] Can you load project home pages?
- [x] Can you load the classification page?
- [x] Can you submit a classification?
- [x] Does talk load correctly?
- [x] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [x] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/talk/discussion-new-form.cjsx [hunk 1]
```diff
@@ -73,6 +76,9 @@ module.exports = createReactClass
       .then (discussion) =>
         @setState loading: false
         titleInput.value = ''
+        @context?.geordi?.logEvent
+          type: 'add-discussion'
+          data: {discussion: discussion.title}
```
Comment (mcbouslog) [inline app/talk/discussion-new-form.cjsx]: unsure if preferred structure, alternatively could be something like 'data: {title: discussion.title}` or something else

Comment (amyrebecca) [reply app/talk/discussion-new-form.cjsx]: just send it either as `data: discussion.title` or else `relatedID: discussion.title`

File: app/talk/discussion.cjsx [hunk 2]
```diff
@@ -171,6 +172,9 @@ module.exports = createReactClass
     @discussionsRequest(discussion)
       .then (discussion) =>
         @setState {discussion: discussion[0]}
+        @context?.geordi?.logEvent
+          type: "talk-view"
+          data: {discussion: discussion?[0]?.title}
```
Comment (amyrebecca) [inline app/talk/discussion.cjsx]: Oh sorry, I meant to say this one needed changed too.

**Comment by mcbouslog** [comment]
This is somewhat related to #4637, noting for reference.

**Pull request merged**